### PR TITLE
Fixed:  bugs in fragment creation

### PR DESCRIPTION
**Pull request by kevinmatthes**
This PR shall fix the bugs in the fragment creation workflows.

### CODE REVIEW

File: .github/workflows/changelog-assembly.yml
Comment (kevinmatthes) [inline .github/workflows/changelog-assembly.yml:1]: This change was suggested by the linter.

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -17,7 +17,7 @@ jobs:
         shell: bash
         if: env.APP_VERSION == ''
         run: |
-          echo "APP_VERSION=${GITHUB_REF#refs/tags/}" >> $GITHUB_ENV
+          echo "APP_VERSION=${GITHUB_REF#refs/tags/}" >> "$GITHUB_ENV"
```
Comment (kevinmatthes) [inline .github/workflows/release.yml:20]: This change was suggested by the linter.

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -68,11 +68,11 @@ jobs:
           if [ "${{ matrix.os }}" = "windows-2022" ]; then
             cp "target/${{ matrix.target }}/release/tjournal.exe" "$staging/"
             7z a -tzip "$staging.zip" "$staging"
-            echo "ASSET=$staging.zip" >> $GITHUB_ENV
+            echo "ASSET=$staging.zip" >> "$GITHUB_ENV"
```
Comment (kevinmatthes) [inline .github/workflows/release.yml:71]: This change was suggested by the linter.

File: .github/workflows/release.yml [hunk 3]
```diff
@@ -68,11 +68,11 @@ jobs:
...
           else
             cp "target/${{ matrix.target }}/release/tjournal" "$staging/"
             tar czf "$staging.tar.gz" "$staging"
-            echo "ASSET=$staging.tar.gz" >> $GITHUB_ENV
+            echo "ASSET=$staging.tar.gz" >> "$GITHUB_ENV"
```
Comment (kevinmatthes) [inline .github/workflows/release.yml:75]: This change was suggested by the linter.

File: .github/workflows/release.yml [hunk 4]
```diff
@@ -28,7 +28,7 @@ jobs:
         with:
           tag_name: ${{ env.APP_VERSION }}
           release_name: TUI-Journal ${{ env.APP_VERSION }}
-          body: Please See [CHANGELOG.md](https://github.com/AmmarAbouZor/tui-journal/blob/main/CHANGELOG.md) for details.
+          body: Please See [CHANGELOG.ron](https://github.com/AmmarAbouZor/tui-journal/blob/main/CHANGELOG.ron) for details.
```
Comment (kevinmatthes) [inline .github/workflows/release.yml:31]: I adjusted the path to the CHANGELOG; the extension was not correct.

File: changelog.d/20230609_072409_Kevin_Matthes_comment-changes.ron
Comment (kevinmatthes) [inline changelog.d/20230609_072409_Kevin_Matthes_comment-changes.ron:1]: I added the fragment for my last two PRs.  This PR should hopefully activate the fragment creation CI such that future PRs will be documented automatically.

File: .github/workflows/comment-changes.yml [hunk 5]
```diff
@@ -12,8 +13,8 @@ permissions:
 jobs:
   aeruginous:
     if: |
-      !contains(github.event.head_commit.message, '[Aeruginous]') && \
-      !contains(github.event.head_commit.message, 'Chore:') && \
+      !contains(github.event.head_commit.message, '[Aeruginous]') &&
+      !contains(github.event.head_commit.message, 'Chore:') &&
```
Comment (kevinmatthes) [inline .github/workflows/comment-changes.yml:17]: The line continuations seem to have been the problem.

File: .github/workflows/main.yml
Comment (kevinmatthes) [inline .github/workflows/main.yml:1]: At the moment, we use GitHub's Super Linter.  This is a Docker image containing various linters for multiple languages.  As we only make use of the GHA linter, at the moment, I will suggest switching to https://github.com/devops-actions/actionlint as soon as it has a license as a subsequent PR in order to speed up the validation.

**Review by AmmarAbouZor — APPROVED** [review]
It looks great. I think the linter would make creating and maintaining GitHub Actions much easier and less painful

Thanks for taking the time to do the research and the adjustments on the workflows :100:

**Pull request merged**